hydroshare/settings.py

```diff
@@ -748,35 +748,27 @@ def _(s): return s # noqa
         },
         "handlers": {
             "djangolog": {
-                "level": "DEBUG",
+                "level": "INFO",
                 "class": "logging.handlers.RotatingFileHandler",
                 "filename": "/hydroshare/log/django.log",
                 "formatter": "verbose",
                 "maxBytes": 1024 * 1024 * 15,  # 15MB
                 "backupCount": 10,
             },
             "hydrosharelog": {
-                "level": "DEBUG",
+                "level": "INFO",
                 "class": "logging.handlers.RotatingFileHandler",
                 "filename": "/hydroshare/log/hydroshare.log",
                 "formatter": "verbose",
                 "maxBytes": 1024 * 1024 * 15,  # 15MB
                 "backupCount": 10,
-            },
-            "celerylog": {
```

Comment:
This was just an attempt to resolve an issue I encountered when running the PR tests. I think it is more of a symptom though.